submission_runner.py

```diff
@@ -108,7 +108,10 @@
 flags.DEFINE_integer('num_tuning_trials',
                      20,
                      'The number of external hyperparameter trials to run.')
-flags.DEFINE_string('data_dir', '~/tensorflow_datasets/', 'Dataset location')
+flags.DEFINE_string('data_dir', '~/tensorflow_datasets/', 'Dataset location.')
+flags.DEFINE_string('imagenet_v2_data_dir',
```

Comment:
Why do we need to specify a separate directory for this dataset? Can't we place it with a fixed naming convention within `data_dir`?

Reply:
yeah, I wasn't sure how we wanted to do this, because for imagenet this is the only workload where we really have two datasets (one for train/val and one for test) which come from different sources. the rest use 1 dataset, or with WMT they use two (WMT 14 vs 17) but they are both in the same format via tfds. if we were only using the `tf.data` pipeline _or_ the pytorch one I think we could do this, but I think it will be easier and more explicit to let the user specify it (even if it is more annoying)

Reply:
My reasoning was that we probably want to automate the data downloading process anyway, so for most users we can determine the location of the imagenet test dataset within the user specified data dir anyway. But I'm ok with leaving it as it is for now.